readthedocsext/theme/templates/includes/elements/promotions/ruby-support.html

```diff
@@ -0,0 +1,9 @@
+<div class="ui card">
+  <div class="content">
+    <a class="header" target="_blank" href="https://about.readthedocs.com/blog/2024/03/read-the-docs-loves-ruby/">Read the Docs ❤️ Ruby</a>
+    <div class="description">
+      We are happy to announce support for Ruby is now available as an option as pre-defined building tools (<code>build.tools</code>).
+      Ruby is now officially supported to build project documentation on Read the Docs 🎉
```

Comment:
Theres some redundancy in language here and this can be reduced. Also, I would avoid mentioning individual configuration options like this, as users might not have the context that `build.tools` is a configuration option in our configuration file. There's a lot to explain there, which is best done on the docs page probably.

I'd probably make this more clear by instead focusing on the tools we support through Ruby -- Ruby support alone isn't as exciting probably.

```suggestion
      Does your project use Jekyll, RDoc, or YARD?
      Ruby is now officially supported on Read the Docs as a pre-defined build tool option!
```

Again, a good place for a CTA `Learn how to use Ruby with your project` or similar.